compliant-standards.md

```diff
@@ -0,0 +1,150 @@
+
+> Using a semantically appropriate landmark with a descriptive label, often this may be the same as the subject line.
+
+### 2.4 Make focus visible
+
+Keyboard focus should be visible at all times within the email content.
+
+> Either use default :focus or suitable custom styling.
+
+### 2.5 Allow for content manipulation
+
+A recipient should be able to manipulate the content to a point to make it readable.
+
+> This can include zoom tools, style adjustment tools, and translation tools.
+
+### 2.6 Email markup should not conflict with email client semantics
+
+Email markup should work in a logical semantic structure without interfering with the email client structure.
+
+> Use logical heading structure. Avoid using a `<main>` landmark in the email content as this is likely to already include a <main> landmark.
+
+### 2.7 Define a language
+
+Emails should define a language and language direction for the content. Any changes of language should also be defined in the markup.
+
+> Set lang and dir attributes
+
+### 2.8 Follow WCAG standards
+
+In addition to these recommendations, you should also consult the latest [WCAG standards](https://www.w3.org/WAI/standards-guidelines/wcag/).
+
+## 3. Resilience
+
+Markup should follow [Postel’s Law](https://en.wikipedia.org/wiki/Robustness_principle): “Be conservative in what you send; be liberal in what you accept.”
+
+### 3.1 Only use supported markup
+
+All code should be within the [WHATWG HTML Standard](https://html.spec.whatwg.org/). Any use of non-standard code should follow the guidelines for [valid custom element names](https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name), [custom data-attributes](https://html.spec.whatwg.org/#embedding-custom-non-visible-data-with-the-data-*-attributes), etc.
+
+> Don’t make up custom elements, attributes, styles, etc. as it may interfere with future development of HTML.
+
+### 3.2 Don’t remove supported code
+
+Supported code should remain intact when unsupported code is removed from around or alongside it by a code sanitizer.
+
+> If invalid or unsupported code is removed, the code around it should remain unchanged. Gmail currently removes an entire style block for one mistake.
+
+### 3.3 Maintain structure when sanitising code
+
+Unsupported elements should keep their supported attributes and be replaced with a semantically neutral element or [valid custom element](https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name).
+
+> Replace `<main class="foo">` with `<div class="foo">`
+
+### 3.4 Group code to fallback when not supported
+
+Code that is dependent on support of other code should be grouped in a way that it can fallback without showing broken partial support.
+
+> Issues can arise when `display: flex` is supported but not `flex-wrap`. We can use things like @supports targeting to help group these so they fallback cleanly.
```

Comment:
> We can use things like

Perhaps "approaches" is a better word than "things" here. Not very important to change.